### ISSUE TITLE
Mobile rendering

### ISSUE THREAD

**Issue by cotillardq** [rapsacnz/DatePickerAura#4]
On mobile the calendar rendering is pretty bad if the table is called from an other table with the class slds-max-medium-table--stacked.
All days end up stacking above one another, making the calendar very long and unusable.
Also the calendar ends up being overlapped by all the other rows.

**Comment by rapsacnz** [comment]
Yeah, it wasn't designed for mobile. It might be worthwhile using an @media
to hide the drop down for mobile
